core/base/document.php

```diff
@@ -1809,9 +1817,12 @@ protected function print_elements( $elements_data ) {
 			$scripts_to_queue = [];
 			$styles_to_queue = [];
 
-			global $wp_scripts, $wp_styles;
+			global $wp_scripts, $wp_script_modules, $wp_styles;
+
+			$printed_script_modules = $this->cache_script_module( $wp_script_modules );
```

Comment:
I need to update the class, to support module caching.

We have a challenge here, the new script module method doesn't work 100% in the same way as the script and style enqueue methods.

For the moment I am caching the printed version of the script module tag.